styles.css

```diff
@@ -103,3 +105,307 @@ header {
+  justify-content: center;
+  width: 45px;
+  height: 24px;
+  background: #ebebff;
+  border-radius: 8px;
+}
+
+.skills-container-2 {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 45px;
+  height: 24px;
+  background: #ebebff;
+  border-radius: 8px;
+  margin-left: 8px;
+}
+
+.skills-container-3 {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 89px;
+  height: 24px;
+  background: #ebebff;
+  border-radius: 8px;
+  margin-left: 8px;
+}
+
+.skills-item {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+}
+
+.position-title {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.work-year {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.rounded-bullet {
+  width: 8px;
+  height: 8px;
+  background: #c1c7d0;
+  border-radius: 8px;
+}
+
+.view-project-button {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  padding: 12px;
+  position: absolute;
+  width: 101px;
+  height: 28px;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+}
+
+.view-project-text {
+  width: 124px;
+  height: 24px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #396df2;
+}
+
+.about-myself-section {
+  margin: 0;
+  width: 375px;
+  height: 1263px;
+  background: #fff;
+  border-top-right-radius: 80px;
+}
+
+.about-header {
+  width: 90px;
+  height: 120px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 800;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  padding-left: 24px;
+  padding-right: 24px;
+  padding-top: 20px;
+  margin-bottom: 0;
+}
+
+.about-description {
+  width: 327px;
+  height: 146px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 200;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  padding-left: 24px;
+  padding-right: 24px;
+  margin-top: 0;
+}
+
+.get-resume-text {
+  width: 160px;
+  height: 24px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #396df2;
+}
+
+.get-resume-button {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  padding: 12px;
+  margin-left: 24px;
+  position: absolute;
+  width: 140px;
+  height: 28px;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+}
+
```

Comment:
- I think it will be great for you to implement the hover effect on your buttons as this is one of the project's requirements. Apparently, this is just to  aid user experience 🌸 

For your template, you are required to have this 👇🏽 
![temp1btn](https://user-images.githubusercontent.com/99246359/198727565-a5f84e50-b7ec-4728-b029-fde763421d24.PNG)

I think you can add the following line of code just below your `.get-resume-button` and `.get-resume-text`
```
.get-resume-button:hover {
  background-color: #6070ff;
  box-shadow: 0px 8px 16px rgba(64, 83, 252, 0.24);
  border-radius: 8px;
}

.get-resume-text:hover {
  color: #fff;
}

.view-project-button:hover {
 . . .
}

.view-project-text:hover {
  . . .
}

```

However, I would like you to implement the same on your `view-project-button` as well. What do you think about implementing this suggestion?? I believe it will make your portfolio look great 🍷  🍷